public/js/app.js

```diff
@@ -1,45 +1,100 @@
-const displaySongs = function() {
-  const nowplayingImg = document.getElementById("nowplayingImg"),
-    nowplayingTitle = document.getElementById("nowplayingTitle"),
-    nowplayingInfo = document.getElementById("nowplayingInfo"),
-    lastPlayedDiv = document.getElementById("lastplayed"),
-    nowPlayingEndpoint = "https://35.211.223.26/nowplaying",
-    lastPlayedEndpoint = "https://35.211.223.26/lastplayed";
+const nowplayingImg = document.querySelector(".now-playing-img"),
+  nowplayingTitle = document.querySelector(".now-playing-title"),
+  nowplayingInfo = document.querySelector(".now-playing-info"),
+  nowPlayingArtist = document.querySelector(".now-playing-artist"),
+  recentlyPlayedDiv = document.querySelector(".recently-played-container"),
+  toggle = document.querySelector("#toggle"),
+  nowPlayingEndpoint = "https://35.211.223.26/nowplaying",
```

Comment:
`http` for now. The server does not support https atm

Reply:
alright cool, I'll change it back